client/app/actions/HelloReduxActions.jsx

```diff
@@ -0,0 +1,8 @@
+import * as actionTypes from '../constants/HelloReduxConstants';
```

Comment:
This file looks like a normal JS file but the file extension says `.jsx`. Is that correct?